physics/ephemeris_body.hpp

```diff
@@ -68,9 +68,6 @@ using ::std::placeholders::_3;
 constexpr Length pre_ἐρατοσθένης_default_ephemeris_fitting_tolerance =
     1 * Milli(Metre);
 constexpr Time max_time_between_checkpoints = 180 * Day;
-// Below this threshold detect a collision to prevent the integrator and the
-// downsampling from going postal.
-constexpr double mean_radius_tolerance = 0.9;
```

Comment:
Let's keep a tolerance so that falling a little too fast doesn't cause Princpia to vaporize the spacecraft; 0.99 should be good enough (if you are going fast enough that your centre of mass would be 1 % of the way to the centre of the body after one frame, survival is unlikely).